ninja-rocker/src/main/java/ninja/rocker/NinjaRockerTemplate.java

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright (C) 2015 Fizzed, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ninja.rocker;
+
+import com.fizzed.rocker.RenderingException;
+import com.fizzed.rocker.RockerOutput;
+import com.fizzed.rocker.runtime.ArrayOfByteArraysOutput;
+import com.fizzed.rocker.runtime.DefaultRockerTemplate;
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import ninja.Context;
+import ninja.Result;
+import ninja.exceptions.InternalServerErrorException;
+import ninja.utils.ResponseStreams;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * New base class for all Rocker templates targeted for the Ninja Framework.
+ * Permits templates to be directly used as a <code>Renderable</code> in a
+ * Result. Also adds all <i>implicit</i> variables that Ninja's default
+ * FreeMarker template system includes.
+ * 
+ * @author joelauer
+ * @param <T>
+ * 
+ * @author Fizzed, Inc (http://fizzed.com)
+ * @author joelauer (http://twitter.com/jjlauer)
+ */
+public abstract class NinjaRockerTemplate<T extends NinjaRockerTemplate> extends DefaultRockerTemplate<T> {
```

Comment:
How is NinjaRockerTemplate integrated into "normal" Ninja views?

I can do 
```
@option extendsClass=ninja.rocker.NinjaRockerTemplate
```
but is there a way to make that a defaultOption for all templates? Otherwise a user has to add extendsClass=... to all templates...